app/javascript/react/components/MonstersIndex.js

```diff
@@ -1,8 +1,8 @@
-import React, { useState, useEffect} from "react"
+import React, { useState, useEffect, useCallback} from "react"
 import MonsterTile from "./MonsterTile"
-const MonstersIndex = () => {
-
+const MonstersIndex = (props) => {
   const [fetchedMonsters, setFetchedMonsters] = useState([])
+  const [state, updateState] = useState ();
```

Comment:
NOOOOO. this is a bad booboo, though I cant tell if it was because of the work I did with you. remove line 5!